packages/ssz/src/web.ts

```diff
@@ -1,8 +1,7 @@
 /** @module ssz */
 import * as ssz from "./index.js";
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-(function (window: any) {
-  // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
```

Comment:
I really do not like this getting switched to an arrow function.  The whole purpose of the iife is to preserve context and be explicit about what is in scope and what is not...